### PR TITLE
Fix tap on notification

### LINKED ISSUE #3967: Notification problem
Hi,

I have three wallets on copay. I received bitcoins in one of them. I clicked on notification icon. 
Problem: opened the wrong wallet window.

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes #3967

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
ACK

File: src/js/services/profileService.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 'use strict';
 angular.module('copayApp.services')
-  .factory('profileService', function profileServiceFactory($rootScope, $location, $timeout, $filter, $log, lodash, storageService, bwcService, configService, notificationService, pushNotificationsService, isChromeApp, isCordova, gettext, gettextCatalog, nodeWebkit, bwsError, uxLanguage, bitcore) {
+  .factory('profileService', function profileServiceFactory($rootScope, $location, $timeout, $filter, $log, sjcl, lodash, storageService, bwcService, configService, notificationService, pushNotificationsService, isChromeApp, isCordova, gettext, gettextCatalog, nodeWebkit, bwsError, uxLanguage, bitcore) {
 
     var root = {};
     var errors = bwcService.getErrors();
+    var usePushNotifications = isCordova && !isMobile.Windows();
```
Comment (cmgustavo) [inline src/js/services/profileService.js]: `isMobile` is not defined.

**Comment by JDonadio** [comment]
Updated

**Comment by gabrielbazan7** [comment]
ACK !

**Pull request merged**